### PR TITLE
Changed all "name" keys to "title" to obey the schema docs

### PR DESCRIPTION
**Pull request by maciekmm**
http://json-schema.org/latest/json-schema-core.html#rfc.section.4.5

### CODE REVIEW

File: Member_authenticate_response.dtd [hunk 1]
```diff
@@ -3,7 +3,7 @@
 	"id": "https://github.com/route4me/json-schemas/blob/master/Member_authenticate_response.dtd",
 	"title": "Membes authenticate response",
 	"type": "object",
-	"name": "Membes_authenticate_response",
+	"title": "Membes_authenticate_response",
```
Comment (maxkoryukov) [inline Member_authenticate_response.dtd]: Confilict: two title tags

File: Asset_tracking.dtd [hunk 2]
```diff
@@ -3,7 +3,7 @@
 	"id": "https://github.com/route4me/json-schemas/blob/master/Asset_tracking.dtd",
 	"title": "Asset tracking",
 	"type": "object",
-	"name": "Asset_tracking",
+	"title": "Asset_tracking",
```
Comment (maxkoryukov) [inline Asset_tracking.dtd]: Conflict: double `title` key on the same level

**Review by maxkoryukov — CHANGES_REQUESTED** [review]
The final JSON (after approving this PR) will be invalid (duplicate keys)

**Comment by maxkoryukov** [comment]
@maciekmm , simple renaming shall not pass))

THere are several (more than 2) schemes, where `title` already exists. And after renaming this schemes have two titles.

Please, could you fix this?

I will try to fix `ajv` schema compiling (a part of Travis build), to provide schema test

**UPD**: Travis fixed

File: Asset_tracking.dtd (hunk 2 above)
Comment (novapulsar) [reply Asset_tracking.dtd]: Members is spelled wrong, it should be Members_authenticate_response

**Comment by maciekmm** [comment]
@maxkoryukov I fixed pointed issues (I had the changes made already, but for some reason they weren't committed).

**Review by maxkoryukov — APPROVED** [review]
Looks good for me now;)

**Comment by maxkoryukov** [comment]
If you don't mind, I am ready to merge.

@maciekmm , @novapulsar , @Olman62 any thoughts on this?

I am asking, because I am not sure about the current usage-status of this repo. I am going to use it (strict schema validation for **nodeJS SDK**), but I don't know - does anybody utilize it already

**Comment by Olman62** [comment]
it's ok, we used it as a documentation- we didn't use it as schema
validation for classes.

**Comment by maciekmm** [comment]
@maxkoryukov I do utilize it for (currently manual) model generation in Go SDK and plan to expand the usage over to Java repository. IMO the repo needs some polishing (naming consistency) maybe splitting schemas into modules/folders related to a specific part of route4me service, but that's not in the scope of this pull request.

What could go into this PR is completely removing _name_ property from everywhere as it's not in the schema documentation, and AFAIK is useless now due to it being the same as the parent property name.

Another thought on the JSON schema is that it mixes http get parameters, headers and json post bodies without any indication. Maybe resigning from the get params and headers would be a good move (especially for validation). What could be even better is splitting the schemas into json and get parameters i.e. "addressbook_body.dtd" and "addressbook_url.dtd".

**Comment by Olman62** [comment]
ok - it's good Idea. I already though about last point.

As I understand, we need addressbook_responce,td, addressbook_body.dtd and
addressbook_url.dtd splitting.

**Comment by maxkoryukov** [comment]
This is great observation by @maciekmm :

> IMO the repo needs some polishing (naming consistency) maybe splitting schemas into modules/folders related to a specific part of route4me service, but that's not in the scope of this pull request.

I would like to move conversation to route4me/route4me-json-schemas#7

**Pull request merged**